source/includes/harvest/_job_openings.md

```diff
@@ -0,0 +1,122 @@
+#Job Openings
+
+This endpoint is used to managing openings on jobs.
+
+## The job opening object
+
+```
+{
+    "id": 123,
+    "opening_id": "OPENED-1",
+    "status": "open",
+    "opened_at": "2015-11-19T19:53:32.565Z",
+    "closed_at": null,
+    "application_id": null
+},
+{
+    "id": 123,
+    "opening_id": "CLOSED-1",
+    "status": "closed",
```

Comment:
@tdphillipsjr Is it significantly more complicated to add close reason in here too?

Reply:
Probably not, but this already shipped.  Add a card to add it and I'll do it while I'm waiting for salman to test post or patch?